docs/monitor-kubernetes.md

```diff
@@ -0,0 +1,305 @@
+# Monitor Kubernetes 
+
+Monitoring the state of the database is crucial to timely identify and react to performance issues. [Percona Monitoring and Management (PMM) solution enables you to do just that](monitoring.md).
+
+However, the database state also depends on the state of the Kubernetes cluster itself. Hence it’s important to have metrics that can depict the state of the Kubernetes cluster.
+
+This document describes how to set up monitoring of the Kubernetes cluster health. This setup has been tested with the [PMM server](https://docs.percona.com/percona-monitoring-and-management/details/architecture.html#pmm-server) as the centralized data storage and the Victoria Metrics Operator as the statistics collector, though the steps should also apply to other monitoring applications. 
+
+## Considerations
+
+1. In this setup we use [Victoria Metrics kubernetes monitoring stack](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack) Helm chart. When customizing the chart's values, consider the following:
+
+    * Since we use the PMM server for monitoring, there is no need to store the data in Victoria Metrics Operator. Therefore, set the `vmsngle.enabled` and `vmcuster.enabled` parameters in the Victoria Metrics Helm chart to `false`.
+    * The Prometheus node exporter is not installed by default since it requires privileged containers with the access to the host file system. If you need the metrics for Nodes, enable the Prometheus node exporter by setting the `prometheus-node-exporter.enabled` flag in the Victoria Metrics Helm chart to `true`.
+    * [Check all the role-based access control rules provided by the charts](https://helm.sh/docs/topics/rbac/) and modify them based on your requirements. 
```

Comment:
We can convey to check RBAC of the victoria metrics k8s stack chart and the dependencies chart like this https://github.com/VictoriaMetrics/helm-charts/blob/ccbb7ae8fdbd952eca3dc2c95e305b13f464d028/charts/victoria-metrics-k8s-stack/Chart.yaml#L34

One of the example is going through the cluster rol of victoria metrics operator, but I believe we should be good in specifying generalized statement like above.
https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-operator/templates/cluster_role.yaml

Reply:
Could you please explain here? Do you mean we should replace the link with the one you provided?